src/mixed-line-bar-chart/bar-series.tsx

```diff
@@ -67,23 +77,30 @@ export default function BarSeries<T extends ChartDataTypes>({
     const PADDING = 4;
     const MINWIDTH = 4;
 
-    if (!stackedBarOffsets && totalSeriesCount > 1) {
+    if (!isStacked && totalSeriesCount > 1) {
       // Regular grouped bars
       barWidth = (barWidth - (totalSeriesCount - 1) * PADDING) / totalSeriesCount;
       barWidth = Math.max(barWidth, MINWIDTH);
     }
 
     return xPoints.map((x, i) => {
       const d = series.data[i];
+      const key = getKeyValue(d.x);
       let barX = x;
       let yValue = d.y;
-
-      if (stackedBarOffsets) {
-        // Stacked bars
-        const offsetMap = d.y < 0 ? stackedBarOffsets.negativeOffsets : stackedBarOffsets.positiveOffsets;
-        yValue = d.y + (offsetMap.get(getKeyValue(d.x)) || 0);
-      } else if (totalSeriesCount > 1) {
-        // Regular grouped bars
+      let isMin = !isStacked;
+      let isMax = !isStacked;
```

Comment:
If there are multiple isMin or isMax for a single row, this is confusing. 
In this case, `isRoundedStart` `isRoundedEnd` is a better name.